src/web-ui/package.json

```diff
@@ -41,6 +41,8 @@
   "resolutions": {
     "acorn": "^6.4.1",
     "elliptic": "^6.5.3",
+    "highlight.js": ">=10.4.1",
```

Comment:
do you need to force these ? is it not induced by amplify?
as suggested I would reduce to minimum versioning

Reply:
Yes, for highlight.js. The immer issue can be resolved by upgrading aws-amplify.